src/qtiCommonRenderer/tpl/choices/simpleChoice.choiceInteraction.tpl

```diff
@@ -22,7 +22,7 @@
             {{/if}}
         </label>
         <div class="label-box">
-            <div class="label-content clear" contenteditable="false" id="choice-{{interaction.serial}}-{{attributes.identifier}}">
+            <div style="width: 100%;" class="label-content clear" contenteditable="false" id="choice-{{interaction.serial}}-{{attributes.identifier}}">
```

Comment:
Can't you avoid inline style? This is not allowed by our coding style.

Reply:
Moreover, I'm not sure about it.
I think it will cause the underlying `img` to take the whole width of the interaction due to the following style rule.
```css
.qti-item .qti-choiceInteraction .choice-area .label-content .qti-block div img {
    width: 100%;
}
```

Kindly double-check your solution.